src/less/section-title/base/section-title.less

```diff
@@ -0,0 +1,225 @@
+/* stylelint-disable no-descending-specificity */
+
+.section-title {
+    display: flex;
+    margin: 30px 0 10px 16px;
+}
+
+.section-title__title {
+    max-width: 75%;
+
+    > h2 {
+        margin: 0;
+    }
+}
+
+.section-title__subtitle {
+    color: @color-grey5;
+}
+
+.section-title__cta {
+    margin: 0 16px;
+    white-space: nowrap;
+
+    > a {
+        border-left: 2px solid @color-grey2;
+        display: inline-block;
+        padding-left: 16px;
+    }
+}
+
+.section-title__cta--no-text {
+    margin: 0;
+
+    > a {
+        border: 0;
+        padding: 0;
+    }
+
+    > a > span:not(.section-title__cta-icon) {
+        display: none;
+    }
+}
+
+.section-title__cta > a,
+.section-title__title > h2 > a {
+    color: @color-black;
+    text-decoration: none;
+
+    &:hover {
+        color: @color-black;
+        text-decoration: underline;
+    }
+
+    &:visited {
+        color: @color-black;
+    }
+}
+
+.section-title__cta-icon {
+    display: inline-block;
+    margin-left: 8px;
+
+    span& {
+        .background-icon-image-mixin('arrow-right-bold', 14px, 14px);
+    }
+
+    svg& {
+        height: 14px;
+        width: 14px;
+    }
+}
+
+.section-title__info {
+    margin: 0 24px 0 8px;
+
+    button.icon-btn.infotip__host {
+        height: 28px;
+        vertical-align: middle;
+        width: 28px;
+    }
+}
+
+.section-title__overflow {
+    margin: 0 24px 0 auto;
+
+    button.icon-btn {
+        height: 28px;
+        width: 28px;
+    }
+}
+
+.section-title__cta,
+.section-title__info,
+.section-title__overflow {
+    align-items: center;
+    display: flex;
+}
+
+@media (max-width: 600px) {
```

Comment:
We should do mobile first. So the default styles should be mobile, and we should override the styles to be on dweb. 
Basically just remove the media query, add these styles to the base ones, and then the dweb one will take precedence on larger screens
(And see if we can combine any)

Reply:
+1 for mobile first

Reply:
done